VMD_scripts/blobulation.tcl

```diff
@@ -532,7 +643,7 @@ proc ::blobulator::blobIndex { blob } {
 	
 	
 	set blobChar q
-	set count 1
+	set count 0
```

Comment:
why this change?

Reply:
This was causing an issue with blob ID's. Off by 1 error.

Reply:
Ah, one too many.